### ISSUE TITLE
Alt-Charset input via custom switch wiring?

### ISSUE THREAD

**Issue by 6800legacy** [ThorstenBr/A2DVI-Firmware#13]
Hi I would like to try adding a switch to select modes with the new 1.3 fw, I have a US IIe enhanced with no switch jumper on the board, so to connect a 2 wire switch one wire goes to the altchar pin on the a2dvi, where do you connect the second wire of the switch? I assume it just needs 5v is there a convenient spot to tap into on the card itself?   

I did noticed the 2 solder pads on the back - "sync" and "altchar" do these need to be soldered? what are these used for?

Also the Config utility seems to hang now after saving configuration to flash, and pressing esc it just goes back to the menu and hangs there?

Thanks for your help.

**Comment by ThorstenBr** [comment]
Yes, close the ALTCHAR solder pad. Ignore the SYNC solderpad (it must remain open). That is the only supported combination. The pads were there because we were still thinking about other options initially. But we'll remove the solderpads with the next PCB update (and ALTCHAR closed, SYNC open will be fixed).

The ALTCHAR input pin has a pull-down resistor on the A2DVI card. It should be wired to switch - and the other switch contact should go +5V. This basically retro fits a "language rocker switch" to your US Apple //e - just like Apple did it with the Euro-Machines.

I would connect the switch to +5V somewhere inside the Apple. Then you can plug the contact to the A2DVI card using a single pin. Probably nicer than soldering wires directly to the A2DVI card.

If possible, consider connecting the switch to +5V through a resistor (maybe something like 1K). Just for safety. If a switch contact somehow gets contact with something else, then you're not going to create a short with the full maximum current of the Apple II power supply. Apple itself did not use a resistor. They indeed directly used +5V for the switch. But their switch was safely installed below the keyboard.

**Comment by 6800legacy** [comment]
Thanks for the update, switch modes working great :-)
